### ISSUE TITLE
Problems with tutorials

### ISSUE THREAD

**Issue by merspieler** [it0uchpods/IDG-A32X#33]
### Expected Behavior
Tutorials guide through the checklists

### Current Behavior
It hangs on some places:
* `Before Start checklist`: hangs on `SIGNS` -> `ON/AUTO`
* `Taxi/Before takeoff checklist`: hangs on `ECAM MEMO` -> `TO NO BLUE`
* `Approach checklist`: hangs on `SEAT BELTS` -> `ON`
* `Landing checklist`: hangs on `ECAM MEMO` -> `LDG NO BLUE`
* `After landing checklist` hangs on `SPOILERS` ->`DISARMED`
* `Parking checklist` hangs on `APU BLEED` -> `ON` but only if APU is OFF -> EXT PWR is used  
If it passes `APU BLEED` it hangs on `ENGINES` -> `OFF`
* `Securing the aircraft checklist` hangs on `ADIRS` -> `OFF`

As well, `AS RQRD` should accept all possible states.

### Steps to Reproduce
1. `Menu` -> `Help` -> `Tutorials`
2. Selec a tutorial and klick `Start Tutorial`
3. You can take `Menu` -> `Help` -> `Aircraft Checklists` as reference (Checks if things can be checked off are on some items broken as well)
4. Go through the checklist and check all items (Use the cockpit, not the `>` in the checklist window.

**Comment by legoboyvdlp** [comment]
Are these the Checklist: X tutorials? In that case they are very buggy and it is difficult to fix this. I didn't think we had any other tutorials apart from the automatically created ones.

**Comment by merspieler** [comment]
yes... these are the tutorials.... and they don't seem to be autogenerated... since they contain specific things like `ECAM MEMO` etc.

**Comment by legoboyvdlp** [comment]
No - they definitely are autogenerated from the A32X-checklists.xml file.

**Comment by merspieler** [comment]
ok... i've reworked them... but i didn't touch any detections

**Comment by legoboyvdlp** [comment]
That most likely did break them, if the conditions for the checklist to be true are not met. 

For example, when you run the landing tutorial, is the landing memo active?

**Comment by merspieler** [comment]
if you can tell me, which files to touch, i could try to fix it

**Comment by legoboyvdlp** [comment]
A32X-checklist.xml, as I already said :wink:

**Comment by legoboyvdlp** [comment]
Don't add conditions until the systems are simulated please.

**Comment by merspieler** [comment]
I can't add them :D

**Comment by merspieler** [comment]
One thing i forgot... which prop contains the state of the `FLAP 3` pb?

**Comment by legoboyvdlp** [comment]
Check the overhead panel pick animation, should be in there somewhere? 

If you can't find it @it0uchpods should know.

**Comment by merspieler** [comment]
Everything on the checklist side, except the above mentioned TODOs, is complete but the TUTORIALS still hang at some points... This is out of my reach.

PR of the updated checklist will come soon™

**Comment by it0uchpods** [comment]
What is the TUTORIALS??? We do not have any, do we? (Why do I not know this, xD)

Flap 3 button is `instrumentation/mk-viii/inputs/discretes/momentary-flap-3-override`

**Comment by merspieler** [comment]
I've found the prop meanwhile... for tutorial: Menu->Help->Tutorials

**Comment by legoboyvdlp** [comment]
Those tutorials are automatically generated, broken, and buggy tutorials @it0uchpods. They are made automatically from the checklist files.

**Comment by it0uchpods** [comment]
Well, I guess we'll write our own tutorials then. Checklists shouldn't contain every step... just the checklist items.

Thanks guys.

**Comment by merspieler** [comment]
They do contain only the checklist so far
